### ISSUE TITLE
linux-gaming: LatencyFleX

### ISSUE THREAD

**Issue by dewdpol** [ventureoo/ARU#44]
Наткнулся на новость о снижении задержки с открытым исходным кодом, как альтернатива NVIDIA Reflex, однако ещё не успел проверить и ознакомиться  https://github.com/ishitatsuyuki/LatencyFleX

**Comment by ventureoo** [comment]
Да, я видел. Но пока не разобрался как именно его использовать. Предлагаю оставить на будущее, когда проект будет хорошо протестирован.

**Comment by ventureoo** [comment]
Итак, из того что я знаю о LatencyFleX, у него есть три случая использования:

1. Использование вместе с Proton NVAPI, **но только для игр которые имеют поддержку NVIDIA Reflex.** 
2. Встраивание в движок UE4, **но только для нативных игр под Linux**
3. Встраивание в движок Unity через BepInEx Bleeding Edge, но работает и для нативных игр под Linux, и для игр запускаемых через Proton.

Лично мне хотелось бы рассмотреть третий случай использования, потому что он наиболее применим к большему количеству игр.

Однако, как видно выше, его использование пока сильно ограничено, и работает он не для всех игр. Непонятно так же как его использовать вместе с обычным Wine.

**Comment by KuKuRuZ1337** [comment]
Бесмысленная вещь, ведь его будет реально использовать только в игрушках, где есть NVIDIA Reflex, но собственно зачем нужен LatencyFleX, когда есть сам NVIDIA Reflex ?
А вот чёрт его знает, нигде не видел, что он даёт задержки меньше, чем NVIDIA Reflex.

**Comment by ventureoo** [comment]
> Бесмысленная вещь, ведь его будет реально использовать только в игрушках, где есть NVIDIA Reflex, но собственно зачем нужен LatencyFleX, когда есть сам NVIDIA Reflex ? А вот чёрт его знает, нигде не видел, что он даёт задержки меньше, чем NVIDIA Reflex.

Где до этого в линуксе была реализация NVIDIA Reflex? Её не было, а LatencyFleX как раз её сделал. LatencyFleX стоит внимания также хотя бы потому что умеет работать и под видеокартами AMD.
